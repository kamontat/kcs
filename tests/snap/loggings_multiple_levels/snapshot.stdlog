[INF] loggings.command     hello info message
[WRN] loggings.command     hello warn message
