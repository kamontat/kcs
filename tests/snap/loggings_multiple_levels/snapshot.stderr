00:10:45 [WRN] cmd.loggings : hello warn message
